pkg/console/validator.go

```diff
@@ -41,6 +41,8 @@ var (
 	ErrMsgNetworkMethodUnknown = "unknown network method"
 
 	ErrMsgSystemSettingsUnknown = "unknown system settings: %s"
+
+	ErrMsgManagementInterfaceNotFound = "`networks` is deprecated, please use `management_interface` for new config and refer https://docs.harvesterhci.io/v1.1/install/harvester-configuration/#installmanagement_interface"
```

Comment:
it seems uncommen to add "`" in code for message, please consider to remove them

Reply:
removed